src/masks.py

```diff
@@ -12,14 +48,24 @@ def get_mask_card_number(card_number: str) -> str:
     Вызывает ошибку:
         ValueError: Если номер карты не состоит из 16 цифр.
     """
-    # Убедимся, что номер карты состоит только из цифр
-    if not card_number.isdigit() or len(card_number) != 16:
-        raise ValueError("Номер карты должен содержать ровно 16 цифр")
+    logger.debug(f"Начало маскирования номера карты: {card_number}")
+
+    try:
+        # Убедимся, что номер карты состоит только из цифр
+        if not card_number.isdigit() or len(card_number) != 16:
+            error_msg = "Номер карты должен содержать ровно 16 цифр"
+            logger.error(f"{error_msg}: {card_number}")
```

Comment:
В логи передаются данные немного по другому
```suggestion
            logger.error("%s: %s", error_msg, card_number)
```
f-строки не используются
https://docs.python.org/3/library/logging.html#logging.Logger.debug